sentry_sdk/scope.py

```diff
@@ -249,6 +252,26 @@ def clear_breadcrumbs(self):
         """Clears breadcrumb buffer."""
         self._breadcrumbs = deque()  # type: Deque[Breadcrumb]
 
+    def add_attachment(
```

Comment:
Should we also maybe provide a method to clear the attachments?

Reply:
There is a general scope clearing method. That might be good enough for a start. If someone asks we can always add more API later.